rfc2822time.go

```diff
@@ -38,6 +40,20 @@ func (t *RFC2822Time) UnmarshalJSON(s []byte) error {
 	return nil
 }
 
+func (t RFC2822Time) GetBSON() (interface{}, error) {
+	return time.Time(t), nil
```

Comment:
`t` is already of `time.Time` type, no? Also, why does it return `interface{}`?

Also, why is it called `get bson`, has a return value `interface` but in fact returns `Time`? :)